### ISSUE TITLE
Link to http://doc.rust-lang.org/complement-bugreport.html broken

### ISSUE THREAD

**Issue by rnestler** [rust-lang/rust#22650]
rustc: provided the following output
```
note: the compiler unexpectedly panicked. this is a bug.
note: we would appreciate a bug report: http://doc.rust-lang.org/complement-bugreport.html
note: run with `RUST_BACKTRACE=1` for a backtrace
```
But the link seems broken:

![screen](https://cloud.githubusercontent.com/assets/1435346/6316457/ca6503fa-ba27-11e4-9505-1f1c94fb3081.png)

The same link also appears on
http://doc.rust-lang.org/#faqs

Here is more of the rustc output: https://gist.github.com/rnestler/9eb208649728ccad0422#file-output

**Comment by steveklabnik** [comment]
Ah yes, this should link to CONTRIBTUING.md now

**Comment by iKevinY** [comment]
It looks like this has been addressed in bb0bbf639eafdd79380a3b1e2e92263a2aa914f9.

**Comment by steveklabnik** [comment]
Awesome, thanks!
